manifests/apache.pp

```diff
@@ -17,7 +17,7 @@
         port                => $::pulp::http_port,
         servername          => $::fqdn,
         serveraliases       => [$::hostname],
-        additional_includes => '/etc/pulp/vhosts80/*.conf',
+        additional_includes => '/etc/httpd/conf.d/vhosts80/*.conf',
```

Comment:
Could you store this base directory in a variable (as we should have in the beginning) and use `$apache::confd_dir` in it? That way other modules can just refer to that variable rather than copying it.